tf/examples/Bonsai/bonsai_example.py

```diff
@@ -74,22 +79,26 @@ def main():
 
     helpermethods.dumpCommand(sys.argv, currDir)
 
+    print ("Num of classes : ",numClasses)
     # numClasses = 1 for binary case
     bonsaiObj = Bonsai(numClasses, dataDimension,
-                       projectionDimension, depth, sigma)
+                       projectionDimension, depth, sigma, isRegression = isRegression)
 
     bonsaiTrainer = BonsaiTrainer(bonsaiObj,
                                   regW, regT, regV, regZ,
                                   sparW, sparT, sparV, sparZ,
                                   learningRate, X, Y, useMCHLoss, outFile)
 
     sess = tf.InteractiveSession()
-    sess.run(tf.global_variables_initializer())
-
+    sess.run(tf.group(tf.initialize_all_variables(),
+                      tf.initialize_variables(tf.local_variables())))
```

Comment:
`tf.initialize_variables()` is deprecated. Please use to `global_variables_initializer()`

Also, tabs vs spaces.